platform/jobs/edxPlatformAccessibilityPr.groovy

```diff
@@ -0,0 +1,149 @@
+    out.println('Exiting with error code 1')
+    return 1
+}
+
+/* Iterate over the job configurations */
+secretMap.each { jobConfigs ->
+
+    Map jobConfig = jobConfigs.getValue()
+
+    /* Test secret contains all necessary keys for this job */
+    /* TODO: Use/Build a more robust test framework for this */
+    assert jobConfig.containsKey('open')
+    assert jobConfig.containsKey('jobName')
+    assert jobConfig.containsKey('repoName')
+    assert jobConfig.containsKey('url')
+    assert jobConfig.containsKey('credential')
+    assert jobConfig.containsKey('cloneReference')
+    assert jobConfig.containsKey('admin')
+    assert jobConfig.containsKey('userWhiteList')
+    assert jobConfig.containsKey('orgWhiteList')
+
+    job(jobConfig['jobName']) {
+
+        /* For non-open jobs, enable project based security */
+        if (!jobConfig['open'].toBoolean()) {
+            authorization {
+                blocksInheritance(true)
+                permissionAll('edx')
+            }
+        }
+        properties {
+              githubProjectUrl(JENKINS_PUBLIC_GITHUB_BASEURL + jobConfig['url'])
+        }
+        parameters {
+            stringParam(params.name, params.default, params.description)
+        }
+        logRotator JENKINS_PUBLIC_LOG_ROTATOR() //discard old builds after 14 days
+        concurrentBuild() //concurrent builds can happen
+        label(JENKINS_PUBLIC_WORKER) //restrict to jenkins-worker
+        scm {
+            git { //using git on the branch and url, clone, clean before checkout
+                remote {
+                    url(JENKINS_PUBLIC_GITHUB_BASEURL + jobConfig['url'] + '.git')
+                    refspec('+refs/pull/*:refs/remotes/origin/pr/*')
+                    if (!jobConfig['open'].toBoolean()) {
+                        credentials(jobConfig['credential'])
+                    }
+                }
+                branch('\${ghprbActualCommit}')
+                browser()
+                extensions {
+                    cloneOptions {
+                        reference('\$HOME/' + jobConfig['cloneReference'])
+                        timeout(10)
+                    }
+                    cleanBeforeCheckout()
+                    relativeTargetDirectory(jobConfig['repoName'])
+                }
+            }
+        }
+        triggers { //trigger when pull request is created
+            pullRequest {
+                admins(jobConfig['admin'])
+                useGitHubHooks()
+                triggerPhrase('jenkins run a11y')
+                userWhitelist(jobConfig['userWhiteList'])
+                orgWhitelist(jobConfig['orgWhiteList'])
+                extensions {
+                    commitStatus {
+                        context('jenkins/a11y')
+                    }
+                }
+            }
+        }
+        wrappers { //abort when stuck after 65 minutes, use gnome-terminal coloring, have timestamps at Console
```

Comment:
It would be better to not put the number in the comment. Actually the comment itself doesn't really add much value, I'd drop it. IIRC we intentionally over-commented the first one so that there would be a good example for people to follow, but it's not really necessary for all DSL scripts.